app/views/layouts/_content.html.haml

```diff
@@ -51,16 +45,8 @@
               = render(:partial => 'layouts/saved_report_paging_bar',
                        :locals  => {:pages => @sb[:pages]})
 
-        - unless simulate?
-          .resizer.hidden-xs
-            .resizer-box
-              .btn-group
-                .btn.btn-default.resize-left
-                  %span.fa.fa-angle-left
-                .btn.btn-default.resize-right{:class => sidewidth == 5 ? 'disabled' : ''}
-                  %span.fa.fa-angle-right
-
-      #left_div.resizable.sidebar-pf.sidebar-pf-left.max-height.scrollable{:class => sidebar}
+      
+      #left_div.sidebar-pf.sidebar-pf-left.scrollable.max-height{:class => simulate? ? 'col-sm-5 col-md-4 col-sm-pull-7 col-md-pull-8' : 'col-sm-4 col-md-3 col-sm-pull-8 col-md-pull-9'}
```

Comment:
What about?
```haml
      - if simulate?
        #left_div.sidebar-pf.sidebar-pf-left.scrollable.max-height{:class => 'col-sm-5 col-md-4 col-sm-pull-7 col-md-pull-8'}
          #default_left_cell
            = yield :left
      - else
        #left_div.sidebar-pf.sidebar-pf-left.scrollable.max-height{:class => 'col-sm-4 col-md-3 col-sm-pull-8 col-md-pull-9'}
          #default_left_cell
            - if @accords && @trees
              = render :partial => "layouts/listnav"
            = yield :left
          #custom_left_cell
```

We don't need trees for simulate and it also eliminates the long line problem...